src/internal/packager/helm/post-render.go

```diff
@@ -141,23 +141,23 @@ func (r *renderer) Run(renderedManifests *bytes.Buffer) (*bytes.Buffer, error) {
 		namespace := rawData.GetNamespace()
 		if _, exists := r.namespaces[namespace]; !exists && namespace != "" {
 			// if this is the first time seeing this ns, we need to track that to create it as well
-			r.namespaces[namespace] = r.options.Cluster.NewZarfManagedNamespace(namespace)
+			r.namespaces[namespace] = r.options.cluster.NewZarfManagedNamespace(namespace)
```

Comment:
I dislike `*Helm` as `options`, its misleading and confusing.

I think composing `*Helm` into `renderer` is prob better

Reply:
Yeah it is a bit easier to read